ProductsWebApi/ProductsWebApi/Repository/UserRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ProductsWebApi.DBContext;
+using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProductsWebApi.Repository
+{
+    public class UserRepository : IUserRepository
+    {
+        private readonly IProductsDBContext _productsDBContext;
+
+        public UserRepository(IProductsDBContext productsDBContext)
+        {
+            _productsDBContext = productsDBContext;
+        }
+
+        public async Task<bool> FetchUserDetails(Users users)
```

Comment:
You are returning only boolean? so method name can be more verbose?
IsValidUser? or something like that